components/push/src/subscriber.rs

```diff
@@ -228,7 +228,7 @@ mod test {
     #[test]
     fn full() -> Result<()> {
         use rc_crypto::ece;
-
+        rc_crypto::ensure_initialized();
```

Comment:
Hrm, so it looks like the `push` crate only calls this in the `decrypt` method in `./src/crypto.rs`. Probably it should also call it in the `generate_key` method as well (which would result in it being initialized by the call to `pm.subscribe` in the test here).